prime-router/src/test/kotlin/fhirengine/engine/fhirRouterTests/DefaultFilterTests.kt

```diff
@@ -172,6 +170,27 @@ class DefaultFilterTests {
         assert(!result)
     }
 
+    @Test
+    fun `test evaluateFilterCondition reverse filter`() {
+        // set up
+        val settings = FileSettings().loadOrganizations(oneOrganization)
+        val one = Schema(name = "None", topic = "full-elr", elements = emptyList())
+        val metadata = Metadata(schema = one)
+
+        val engine = spyk(makeFhirEngine(metadata, settings, TaskAction.route) as FHIRRouter)
+        val bundle = FhirTranscoder.decode(fhirCodeP)
+
+        // act
+        val useDefaultResult = engine.evaluateFilterCondition(emptyList(), bundle, false, true)
+        // assert
+        assert(useDefaultResult)
```

Comment:
Use assertThat instead in all your tests.  This is the program directed unit test framework. Assert is now only used if you cannot use assertThat which is rare.  See assertk

Reply:
Is that requirement meant to improve readability or is there some other reason? Is there any kind of document or decision record that we could reference to have a better idea of these kinds of standards, or is it more of an informal shared understanding at this point?

Reply:
Updated unit tests to user assertThat. Ran unit tests after making these changes and they're all passing.